### ISSUE TITLE
The relationship between B-Spline and Indicator function

### ISSUE THREAD

**Issue by WangYangDi** [mkazhdan/PoissonRecon#239]
Hi,Professor

I have a question about the relationship between B-Spline and Indicator function. Indicator function takes
the value 1 inside the surface and 0 outside the surface.B-spline is the function that define in the each node of 
the tree.how to make Indicator function out of B-spline functions?

**Comment by mkazhdan** [comment]
Indeed. At the end of the day, the indicator function is represented as a linear combination of the B-splines. So that solving for the indicator function amounts to solving for a coefficient at each node, giving the contribution of that node's B-spline to the overall function.

**Comment by mkazhdan** [comment]
I'm afraid I cannot parse that.

**Comment by WangYangDi** [comment]
Thanks

**Comment by WangYangDi** [comment]
Hi,professor

I have a question that how to obtain a triangular grid from the obtained indicative function.
By means of Poisson reconstruction, we can calculate the indicative function of space. If I want a  triangular grid on a surface, how can i achieve it by Poisson reconstruction?

**Comment by mkazhdan** [comment]
Methods like Marching Cubes are designed to extract triangle meshes from implicit functions (represented by voxel grids).

**Comment by WangYangDi** [comment]
Thanks，professor
